assets/js/index.js

```diff
@@ -44,4 +55,40 @@ function clearAddTodoInput() {
 function clearErrorMessage() {
   errorMessage.innerText = "";
   backendErrorMessage.innerText = "";
+}
+
+async function getTasks() {
+  try {
+    const response = await fetch(url);
+    const {data} = await response.json();
+    return data;
+  } catch {
+    backendErrorMessage.innerText = "Something went wrong!";
+  }
+}
+
+function changeDateFormat(createdAt) {
+  const dateTime = new Date(createdAt);
+  const year = dateTime.toLocaleDateString("en-US", { year: "numeric" });
+  const month = dateTime.toLocaleDateString("en-US", { month: "short" });
+  const day = dateTime.toLocaleDateString("en-US", { day: "numeric" });
+
+  return `${day} ${month} ${year}`;
+}
+
+function showTasks(tasks) {
+  const sortedTasks = tasks.sort(
+    (objA, objB) => new Date(objB.createdAt) - new Date(objA.createdAt),
+  );
+
+  let listHtml = "";
+  sortedTasks.forEach(({text, createdAt}) => {
+    listHtml += ` <li class="todo-item">
+    <input type="checkbox" />
+    <p class="todo"  title="${changeDateFormat(createdAt)}">${text}</p>
+    <i class="fa-solid fa-pen-to-square edite-icon"></i>
+    <i class="fa-solid fa-xmark remove-icon"></i>
+    </li>`
+  });
+  todoList.innerHTML = listHtml;
```

Comment:
This is completely fine. Just, it can be written like this as well:
```suggestion
  todoList.innerHTML = tasks.sort(
    (a, b) => new Date(b.createdAt) -  new Date(a.createdAt)
  ).map(({text, createdAt}) => ` <li class="todo-item">
    <input type="checkbox" />
    <p class="todo"  title="${changeDateFormat(createdAt)}">${text}</p>
    <i class="fa-solid fa-pen-to-square edite-icon"></i>
    <i class="fa-solid fa-xmark remove-icon"></i>
    </li>`).join("");
```
Please read through this and try to understand it. You should not change the code unless you understand it completely.

Reply:
Done.